cli/config/configStructs/tapConfig.go

```diff
@@ -79,10 +78,6 @@ func (config *TapConfig) GetInsertionFilter() string {
 			}
 		}
 	}
-	_, err := basenine.Parse(insertionFilter)
```

Comment:
why is this removed?

Reply:
@RoyUP9 yes, the addition of [this method](https://github.com/up9inc/basenine/blob/e2ca51087607e7e2f6688b0c4c1655b9ec552969/server/lib/helpers.go#L38-L71) to the shared library made the build error for Windows target. We don't support Windows in Basenine right now. So, I had to remove that syntax error check from there.

The user should guarantee that there are no syntax errors in the given insertion filter. If there are any, the insertion filter is ignored.

Reply:
@mertyildiran 
I remember that I asked for this addition and we might want to add it in the future but as long as it causes the windows binary to be broken it can be removed (more important to have valid binary and remove the small check).
so I approved this change and it can be merged before the next release.